src/server/pfs/server/driver_new_storage.go

```diff
@@ -343,15 +343,14 @@ func deserializeShard(shardAny *types.Any) (*pfs.Shard, error) {
 func (d *driver) compactionWorker() {
 	ctx := context.Background()
 	w := work.NewWorker(d.etcdClient, d.prefix, storageTaskNamespace)
-
+	// Configure backoff so we retry indefinitely
 	backoffStrat := backoff.NewExponentialBackOff()
-	backoffStrat.MaxElapsedTime = 0 // don't give up, compactionWorker should not exit
-
+	backoffStrat.MaxElapsedTime = 0
 	notifier := func(err error, t time.Duration) error {
```

Comment:
We tend to define this inline with the `RetryNotify` call (similar to how the operation is defined). It is not going to get used anywhere else.